code/modules/mob/living/living.dm

```diff
@@ -1127,18 +1127,31 @@ default behaviour is:
 		if ((!( yes ) || now_pushing) || !loc)
 			return
 		now_pushing = 1
-		if (istype(AM, /mob/living))
+		if (istype(AM, /obj/structure/bed/roller)) //no pushing rollerbeds that have people on them
+			var/obj/structure/bed/roller/R = AM
+			for(var/mob/living/tmob in range(R, 1))
+				if(tmob.pulling == R && !(tmob.restrained()) && tmob.stat == 0 && R.density == 1)
+					if (!(world.time % 5))
```

Comment:
You may want to just remove the check altogether and leave it as is in the rest because I'm sure it's almost never worked this whole time and people are just used to it this way by now.